backend/src/services/project/project-service.ts

```diff
@@ -86,19 +91,41 @@ export const projectServiceFactory = ({
   /*
    * Create workspace. Make user the admin
    * */
-  const createProject = async ({ orgId, actor, actorId, actorOrgId, workspaceName, slug }: TCreateProjectDTO) => {
+  const createProject = async ({
+    orgId,
+    orgSlug,
+    actor,
+    actorId,
+    actorOrgId,
+    workspaceName,
+    slug
+  }: TCreateProjectDTO) => {
+    if (orgSlug && orgId) {
```

Comment:
Why do we still support both

Reply:
V1 project router still uses the `orgId`, I'm unsure where this is actually used though?

Reply:
I don't think we actually use it. Perhaps we should just delete the endpoint entirely? @maidul98